src/main/java/com/hillel/items_exchange/security/jwt/JwtTokenProvider.java

```diff
@@ -63,17 +66,21 @@ public String resolveToken(HttpServletRequest req) {
         if (bearerToken != null && bearerToken.startsWith(BEARER_PREFIX)) {
             return bearerToken.substring(7);
         }
+        if (bearerToken != null && !bearerToken.startsWith(BEARER_PREFIX)) {
```

Comment:
You have almost the same check twice here!
You should return Optional instead of null.

Reply:
But you asked me to add this, just for Swagger-UI